s3_website.yml.example

```diff
@@ -1,60 +0,0 @@
-s3_id: YOUR_AWS_S3_ACCESS_KEY_ID
```

Comment:
s3_website.yml was needed to deploy to S3 from command line (and it wasn't supposed to be committed into the repo for security, thus .example). It's no longer required since S3 configuration is moved to the autobuild server